src/utils/deserialize.ts

```diff
@@ -0,0 +1,261 @@
+import { PublicKey } from '@solana/web3.js';
+import {
+  BinaryReader, Schema, BorshError, BinaryWriter,
+} from 'borsh';
+
+(BinaryReader.prototype as any).readPubkey = function () {
+  const reader = this as unknown as BinaryReader;
+  const array = reader.readFixedArray(32);
+  return new PublicKey(array);
+};
+
+(BinaryWriter.prototype as any).writePubkey = function (value: PublicKey) {
+  const writer = this as unknown as BinaryWriter;
+  writer.writeFixedArray(value.toBuffer());
+};
+
+export enum VoteTypeKind {
+  SingleChoice = 0,
+  MultiChoice = 1,
```

Comment:
What is a multichoice vote?

Reply:
Feature of governance that nobody uses. You can propose multiple options instead of a yes/no